addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js

```diff
@@ -50,6 +52,11 @@ export class SpreadsheetDashboardAction extends Component {
             if (activeDashboardId) {
                 this.openDashboard(activeDashboardId);
             }
+            if (this.env.debug) {
+                this.isDashboardAdmin = await user.hasGroup(
```

Comment:
We could do a pormise.all with the `load` to do the RPC in parallel. Idk if we care enough tho

Reply:
I believe using `Promise.all` here could reduce code readability, as we'd need to conditionally make the request only in debug mode. This would require wrapping the `Promise.all` in an `if-else` block, which might complicate the logic.

Reply:
Yeah I totally agree, that's why I was wondering if we wanted to do it :shrug: